pkgs/by-name/on/onionbalance/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  python313Packages,
+  fetchFromGitLab,
+}:
+
+python313Packages.buildPythonPackage rec {
+  pname = "onionbalance";
+  version = "0.2.4";
+  src = fetchFromGitLab {
+    domain = "gitlab.torproject.org";
+    owner = "tpo";
+    repo = "onion-services/onionbalance";
+    rev = "36938bf60cc71745fc9ac7b44a79ec9c06ac620e";
+    sha256 = "sha256-amwKP9LJ7aHPECNUNTluFpgIFSRLxR7eHQxBxW5574I=";
```

```suggestion
    hash = "sha256-amwKP9LJ7aHPECNUNTluFpgIFSRLxR7eHQxBxW5574I=";
```
use hash for a sri hash